scripts/restart_app.sh

```
@@ -1,4 +1,4 @@
 docker stop chanakya
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)